pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -544,7 +544,6 @@ let
       CGROUP_RDMA    = whenAtLeast "4.11" yes;
 
       MEMCG                    = yes;
-      MEMCG_SWAP               = yes;
```

Comment:
You want to say `whenOlder "6.1" yes` instead of just removing it.

Reply:
done